modules/op-release-notes-1-8.adoc

```diff
@@ -12,14 +12,279 @@ With this update, {pipelines-title} General Availability (GA) 1.8 is available o
+// link:https://github.com/tektoncd/hub/pull/542
+// Shiv Pratap Verma @pratap0007
+
+* With this update, you no longer need to log in to the {tekton-hub} web console to add resources from the catalog to the database. Now, these resources are automatically added when the {tekton-hub} API starts running for the first time.
+// link:https://github.com/tektoncd/hub/pull/510
+// Shiv Pratap Verma @pratap0007
+
+* This update automatically refreshes the catalog every 30 minutes by calling the catalog refresh API job. This interval is user-configurable.
+// link:https://github.com/tektoncd/hub/pull/547
+// Shiv Pratap Verma @pratap0007
+
+[id="pac-new-features-1-8_{context}"]
+=== {pac}
+// {pac} owners: Shivam Mukhade Savita Ashture
+
+:FeatureName: {pac} (PAC)
 include::snippets/technology-preview.adoc[]
 
+* With this update, as a developer, you get a notification from the `tkn-pac` CLI tool if you try to add a duplicate repository to a {pac} run. When you enter `tkn pac create repository`, each repository must have a unique URL. This notification also helps prevent hijacking exploits.
+// link:https://issues.redhat.com/browse/SRVKP-1758[SRVKP-1758 Pipeline as code: validate repository URL uniqueness])
+// Shivam Mukhade
+
+* With this update, as a developer, you can use the new `tkn-pac setup cli` command to add a Git repository to {pac} by using the webhook mechanism. This way, you can use {pac} even when using GitHub Apps is not feasible. This capability includes support for repositories on GitHub, GitLab, and BitBucket.
+// link:https://issues.redhat.com/browse/SRVKP-2137[SRVKP-2137 Pipeline as code - CLI add repository via webhooks])
+// Shivam Mukhade
+
+* With this update, {pac} supports GitLab integration with features such as the following:
+** ACL (Access Control List) on project or group
+** `/ok-to-test` support from allowed users
+** `/retest` support.
+//For more information, https://pkg.go.dev/github.com/openshift-pipelines/pipelines-as-code/pkg/provider/gitlab#section-readme
+// no link
+
+* With this update, you can perform advanced pipeline filtering with Common Expression Language (CEL). With CEL, you can match pipeline runs with different Git provider events by using annotations in the `PipelineRun` resource. For example:
++
+[source,yaml]
+----
+  ...
+  annotations:
+     pipelinesascode.tekton.dev/on-cel-expression: |
+      event == "pull_request" && target_branch == "main" && source_branch == "wip"
+----
+// no link
+
+* Previously, as a developer, you could have only one pipeline run in your `.tekton` directory for each Git event, such as a pull request. With this update, you can have multiple pipeline runs in your `.tekton` directory. The web console displays the status and reports of the runs, and the reports include comments.
```

Comment:
this point and the point below`With this update, {pac} supports multiple pipeline...` are same 
we can merge them 
something like below
```
Previously, as a developer, you could have only one pipeline run in your .tekton directory for each Git event, such as a 
pull request. With this update, you can have multiple pipeline runs in your .tekton directory. The web console displays the
status and reports of the runs. The pipeline runs operate in parallel and report back to the Git provider interface.
```